documents/moving-across-committees.md

```diff
@@ -0,0 +1,25 @@
+# Transfering a Working Group from the Node.js TSC to the Node.js Community Committee
+
+As interested, Working Groups that are focused on serving, growing, and building the community surrounding Node.js may decide that it would like to become a part of the Node.js Community Committee. Similarly, if a Working Group under the Node.js Community Committee is focused on more technical tasks and may want to transfer over to the TSC.
+
+This document is a guide on how to begin the transfer process from one committee to another.
+
+## Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
+  * **Step 1.1**: If there are no objections, continue on to step two.
+* **Step 2**: Create an issue in the repository for the Working Group in question, proposing the move change. Given the governance model, this will likely be a vote and will need majority approval of the Working Group's members.
```

Comment:
I think this should be step one, not step two. Asking to move a WG before asking if that's something the WG wants to do has the potential to cause a lot of conflict and confusion.